pkg/relic/Makefile

```diff
@@ -10,14 +10,18 @@ endif
 .PHONY: all
 
 all: $(PKG_BUILDDIR)/Makefile
-	"$(MAKE)" -C $(PKG_BUILDDIR) && \
-	cp $(PKG_BUILDDIR)/lib/librelic_s.a $(BINDIR)$(PKG_NAME).a
+	@echo "Building relic package..." ; \
+		OUTPUT="$$("$(MAKE)" -C $(PKG_BUILDDIR) 2>&1)" || \
+		{ echo $$OUTPUT; false; } && \
```

Comment:
We could also use `$(AD)` here, so that one would see the output when building with `QUIET=0`.